src/css/betterttv.css

```diff
@@ -2347,6 +2350,8 @@ body.channel_new.columns.ember-application {
     cursor: pointer;
 }
 
-.chat-line .bttv-deleted {
+.chat-line .bttv-deleted.bttv-click {
     opacity: 0.5;
+    text-decoration: none;
```

Comment:
This only affects the opacity of messages with `bttv-click`. With 'Show Delete Messages' enabled the class isn't added, showing the normal message with no opacity.